src/pages/NewGroupPage.js

```diff
@@ -134,6 +134,9 @@ class NewGroupPage extends Component {
      * @param {Object} option
      */
     toggleOption(option) {
+        // After toggling an option, refocus the search bar to help find the next selection
+        this.optionSelector.textInput.focus();
```

Comment:
While this works, it is pretty unusual and we should try to make this call in the direct parent of that text input.

One option could be to use [ref forwarding](https://reactjs.org/docs/forwarding-refs.html). But in this case, I'd actually suggest we just move this logic closer to the text input. Here's what I'd suggest:

1. Create a new method in `OptionsSelector` called `this.selectRow()`
1. In that method, do something like this
```js
selectRow() {
    this.textInput.focus();
    this.props.onSelectRow(...)
}
```
3. Update all usages in `OptionsSelector` to pass `this.selectRow` instead of `this.props.onSelectRow`
4. If you only need this behavior sometimes (since I see we are only adding it to `NewGroupPage` and not other pages) maybe pass a boolean prop like `shouldFocusOnSelectRow` to the `OptionsSelector` ?